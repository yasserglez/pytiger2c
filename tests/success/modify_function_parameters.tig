let
	var n:= 10
	function f(n : int) =
	let
		function g() = n := n + 1
	in
		printi(n); print("\n");
		g();
		printi(n); print("\n")
	end
in
	printi(n); print("\n");
	f(n);
	printi(n)
end