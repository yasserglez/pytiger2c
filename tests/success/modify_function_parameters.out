10
10
11
10